crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -240,11 +248,34 @@ impl CallParachainCommand {
 		Ok(())
 	}
 
+	/// Checks if the chain has the Sudo pallet and prompt the user to confirm if they want to
+	/// execute the call via sudo.
+	async fn configure_sudo(&mut self, chain: &Chain, cli: &mut impl Cli) -> Result<()> {
+		match find_extrinsic_by_name(&chain.pallets, "Sudo", "sudo").await {
+			Ok(_) =>
+				if !self.sudo {
+					self.sudo = cli
+						.confirm(
+							"Would you like to dispatch this function call with `Root` origin?",
+						)
+						.initial_value(false)
+						.interact()?;
+				},
+			Err(_) =>
+				if self.sudo {
+					cli.warning("NOTE: sudo extrinsic is not supported by the chain. Ignoring `--sudo` flag.")?;
```

Comment:
```suggestion
					cli.warning("NOTE: sudo is not supported by the chain. Ignoring `--sudo` flag.")?;
```

Reply:
Improved in https://github.com/r0gue-io/pop-cli/pull/349/commits/620252fefff47e47f786f3ba4f5dbbfeba00bd74